dev/host_vars/is_1.yml

```diff
@@ -19,32 +19,31 @@
 # requirements. If you need to add additional configurations, please use the custom.yml under each role.
 
 hostname: localhost
-mgt_hostname: localhost
-ports_offset: 0
+port_offset: 0
 
 # KeyStore which will be used for encrypting/decrypting passwords and other sensitive information.
-keystore_location: ${carbon.home}/repository/resources/security/wso2carbon.jks
+keystore_location: wso2carbon.jks
 keystore_type: JKS
 keystore_password: wso2carbon
 keystore_key_alias: wso2carbon
 keystore_key_password: wso2carbon
 
 # The KeyStore which is used for encrypting/decrypting internal data. This block is read by Carbon Crypto Service.
-internal_keystore_location: ${carbon.home}/repository/resources/security/wso2carbon.jks
+internal_keystore_location: wso2carbon.jks
 internal_keystore_type: JKS
 internal_keystore_password: wso2carbon
 internal_keystore_key_alias: wso2carbon
 internal_keystore_key_password: wso2carbon
 
 # System wide trust-store which is used to maintain the certificates of all the trusted parties.
-truststore_location: ${carbon.home}/repository/resources/security/client-truststore.jks
+truststore_location: client-truststore.jks
 truststore_type: JKS
 truststore_password: wso2carbon
 
-# Configure proxy ports. Note that this is a optional attribute.
-# catalina_server_connector_http_proxy_port: 80
-catalina_server_connector_http_max_threads: 250
-catalina_server_connector_http_accept_count: 200
-# catalina_server_connector_https_proxy_port: 443
-catalina_server_connector_https_max_threads: 250
-catalina_server_connector_https_accept_count: 200
+## Configure proxy ports. Note that this is a optional attribute.
+## catalina_server_connector_http_proxy_port: 80
+#catalina_server_connector_http_max_threads: 250
+#catalina_server_connector_http_accept_count: 200
+## catalina_server_connector_https_proxy_port: 443
+#catalina_server_connector_https_max_threads: 250
+#catalina_server_connector_https_accept_count: 200
```

Comment:
Any specific reason why we are not using these? Also, remove the double comments (##) on some of these parameters.

Reply:
Fixed in https://github.com/wso2/ansible-is/pull/35/commits/9235b840f82068a121bd7ce0022fa7467548a5a7.